source/index.js

```diff
@@ -63,6 +63,10 @@ class likely {
      * @param {Object} [options] additional options for each widget
      */
     static initiate(nodes, options) {
+        if (!isBrowserEnv) {
+            return;
+        }
```

Comment:
@vitkarpov, thanks for your attention! I didn’t notice this myself.

@effrenus, can you please remove this check?